custom_metrics/privacy.js

```diff
@@ -1,12 +1,43 @@
 //[privacy]
 // Uncomment the previous line for testing on webpagetest.org
 
+// README! Instructions for adding a new custom metric for the Web Almanac.
+// 2. If the value requires more than one line of code, evaluate it in an IIFE, eg `(() => { ... })()`. See `link-nodes`.
+// 3. Test your change by following the instructions at https://github.com/HTTPArchive/almanac.httparchive.org/issues/33#issuecomment-502288773.
+// 4. Submit a PR to update this file.
+
+const response_bodies = $WPT_BODIES;
+
+/**
+ * @function findStringInBodies
+ * inspired by https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/event-names.js
+ *
+ * @param {string} string - String to look for in the response bodies.
+ * @return {boolean} - True, if string was found.
+ */
+function findStringInBodies(string) {
+  try {
+    return response_bodies.some((body) => {
```

Comment:
I'd suggest including a filter on the response bodies so that you only check responses that are likely to contain the input strings. For example, only JS and HTML responses.

Reply:
Filtered on `Document` and `Script` types in ebd97c8. 
`type` looks to me like the only property that allows us to determine whether a response is JS/HTML, or am I missing something?
Could `XHR` and `Fetch` also be sources of relevant HTML/scripts?

Reply:
I think `Document` and `Script` are fine.